pages/participantinfo.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+    Button,
+    Box,
+    Center,
+    Text,
+    Input,
+    FormControl,
+    FormLabel,
+    Progress,
+    Stack,
+    HStack,
+    VStack,
+} from "@chakra-ui/react";
+import { useState } from "react";
+
+/**
+ * This is the page that a user will use to enter the participants personal information
+ * onto the SDC platform as a parent of volunteer
+ */
+export default function Participantinfo(): JSX.Element {
+    const [progressBar, setprogressBar] = useState(Number);
+    if (progressBar <= 0) {
+        setprogressBar(14);
```

Comment:
```suggestion
        setProgressBar(14);
```

Reply:
Why exactly 14? Can you make this into a constant?

Reply:
yea we'll probably change it into a constant we were just testing it with a number to make sure it works. We chose 14 because there were 7 pages so 100/7 is approx 14.